src/invisible/explainers/enrichments/reference-image-mapper.md

```diff
@@ -0,0 +1,211 @@
+    <h4>Scanning video</h4>
+    <Youtube src="0U4H-uOIHlw"/>
+    We take a longer scanning recording, about 2 min. The angles and distances cover what a person might see whilst walking past or standing in front of the building.
+  </div>
+</div>
+
+## Scanning best practices
+
+- Make the recording while holding the Pupil Invisible glasses in your hand rather than wearing it on your head.
+- Record the object of interest from all possible angles and from all distances a subject may look. More diversity is better. Collecting sufficiently diverse viewpoints or a large object, like a building, may take longer than capturing a small object like a magazine.
+- Move the glasses slowly while recording to avoid motion blur.
+- Make sure to have good contrast and that your scene lighting during scanning is similar to that during mapping.
+
+What works and what doesn’t?
+- These examples are **feature rich**. Reference Image Mapper needs enough salient visual content to produce a successful mapping.
+- The scenes have relatively **static** features in the environment. If there is a lot of movement or the objects change in appearance, the mapping *can* fail.
+
+::: tip
+**Ready to go?**
+Why not try replicating the above examples? Or even try it with your own use-cases! If you haven’t already, we recommend you check out the [Cloud Getting Started Guide](/invisible/getting-started/analyse-recordings-in-pupil-cloud/#analyse-recordings-in-pupil-cloud), which covers the basics of working with enrichments.
+:::
+
+## Validate gaze mapping
+
+<div class="pb-4" style="display:flex;justify-content:center;">
+  <v-img
+    class="rounded" 
+    style="margin-bottom:32px;"
+    :src="require('../../../media/invisible/rim/rim-in-cloud.png')"
+    max-width=90%
+  >
+  </v-img>
+</div>
+
+To check if gaze has been mapped successfully, use the side-by-side view:
+1. Select a recording
+2. Select the Reference Image Mapper Enrichment
+3. Select the Scene / Reference Image View
+4. If you want to visualize and evaluate the 3D model generated (white dots), just turn on the Point Cloud toggle!
+
+Now when you play back the recording you can see where gaze is mapped to on your reference image for validation
+
+## Occlusions
+
+<div class="pb-4" style="display:flex;justify-content:center;">
+  <v-img
+    class="rounded" 
+    :src="require('../../../media/invisible/rim/basketball-occlusion.png')"
+    max-width=400px
+  >
+  </v-img>
+</div>
+
+
+Sometimes an object will occlude the feature/object of interest. The reference image mapper may not know this, so a false positive mapping could occur. 
+
+If you need to remove the falsely mapped data points, there are a few workarounds.
+
+1. Use Events and Sections to trim out false positive sections.
+2. Manually remove the affected data points in the reference image mapper export, by finding the timestamp and deleting the row in the .csv export.
+
+## Repetitions
+
+**Use Sections to map portions of your recordings**
+
```

Comment:
```suggestion
```
Given that this comes right after an H2 it seems repetitive to have another header.